controllers/smf/resources_test.go

```diff
@@ -151,16 +152,60 @@ func TestCreateDeployment(t *testing.T) {
 
 func TestCreateConfigMap(t *testing.T) {
 	log := log.FromContext(context.TODO())
+	var refList []*refv1alpha1.ConfigRef
 	smfDeployment := newSmfDeployment("test-smf-deployment")
-	got, err := createConfigMap(log, smfDeployment)
+	ref := &refv1alpha1.ConfigRef{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "free5gc-upf-1",
+		},
+		Spec: refv1alpha1.ConfigRefSpec{
+			GVKC: refv1alpha1.GVKC{
+				Group:   "workload.nephio.org",
+				Version: "v1alpha1",
+				Kind:    "UPFDeployment",
+				Config:  `{"capacity":{"maxUplinkThroughput":"1G","maxDownlinkThroughput":"5G","maxSessions":1000, "maxSubscribers":1000,"maxNFConnections":2000},"interfaces":[{"name":"n6","ipv4":{"address":"10.10.12.10/24","gateway":"10.10.12.1"}},{"name":"n3","ipv4":{"address":"10.10.10.10/24","gateway":"10.10.10.1"}},{"name":"n4","ipv4":{"address":"10.10.11.10/24","gateway":"10.10.11.1"}}],"networkInstances":[{"name":"vpc-internet","interfaces":["n6"],"dataNetworks":[{"name":"apn-test","pool":[{"prefix":"100.100.0.0/16"}]}]}]}`,
```

Comment:
This is not aligned with the global topology used where we are using subnet in 172 ranges

Reply:
@debeaueric 

That is unit test code. For e2e, the configref for SMFDeployment CR will be generated by specialization code

Reply:
ok